.github/workflows/browser-tests.js.yml

```
@@ -0,0 +1,18 @@
+name: Browser tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  browser-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4.1.6
+      - run: cp .env.development.prod .env
+      - run: sudo chown -R 1000:1000 ${GITHUB_WORKSPACE}
+      - run: docker compose run ui npm install
```

```suggestion
      - run: docker compose run --rm ui npm install
```
No need to keep the dangling container post-install.